### ISSUE TITLE
Corporate Crime

### ISSUE THREAD

**Issue by castorsia** [jsoma/data-studio-projects#254]
# Pitch

## Summary

Since corporations have the same rights as people do, let's look at the patterns of their criminality. By using data from the Corporate Prosecution Registry of the University of Virginia School of Law, I intent to ask the following questions:

-Who are the repeat offenders?
- What types of crimes corporations commit?
-What is the economic impact on them?
-Who gets prosecuted under the Foreign Corrupt Practises Act, the most? US or foreign companies?  
-Arms, drugs and all the fun stuff.

![2018-08-06 18_13_24-project_4](https://user-images.githubusercontent.com/39416165/43743850-69c1e4ee-99a5-11e8-8d90-6c0fee2dd7e8.png)

![2018-08-06 18_14_10-project_4](https://user-images.githubusercontent.com/39416165/43743851-6bacc6ac-99a5-11e8-8c71-b99e45483885.png)

## Details

**Possible headline(s):** 

**Data set(s):** (http://lib.law.virginia.edu/Garrett/corporate-prosecution-registry/browse/browse.html)

**Code repository:** (https://github.com/castorsia/data-studio/commit/7edc1adfdd9ebb23e281beaea39f8bb5b3430b21)
	
**Possible problems/fears/questions:** 

Too categorical of a dataset. Hence not easily visualised. But interesting leads.

## Work so far

Explored my dataset and made a couple of graphs.
Found leads for new stories (arms, fraud and drug).

## Checklist

This checklist must be completed before you submit your draft.

- [X ] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [X ] My issue links to my data set(s)
- [ X] My issue links to my code repository
- [ X] My issue explains what I'd like to explore in the data set
- [ X] My issue includes images - either inspiration or what I've done so far

**Comment by sarahslo** [comment]
sounds like the categories are too broad? is that what you are saying? i like the idea in general, but
i'm not sure about this universe of data. what exactly are they counting that the numbers are so small?

**Comment by adrianblanco** [comment]
Agree! Are there any more specific categories? Your story will be more powerful with more specific examples or cases.
Also, I am interested in what is the economic impact. Will you use any other data source to measure it?
